financial-services-accelerator/components/org.wso2.financial.services.accelerator.consent.mgt.extensions/src/main/java/org/wso2/financial/services/accelerator/consent/mgt/extensions/job/scheduler/PeriodicalConsentJobScheduler.java

```diff
@@ -0,0 +1,111 @@
+/**
+ * Copyright (c) 2025, WSO2 LLC. (https://www.wso2.com).
+ * <p>
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.financial.services.accelerator.consent.mgt.extensions.job.scheduler;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.quartz.Scheduler;
+import org.quartz.SchedulerException;
+import org.quartz.impl.StdSchedulerFactory;
+import org.wso2.carbon.utils.CarbonUtils;
+import org.wso2.financial.services.accelerator.common.util.Generated;
+
+import java.io.File;
+import java.nio.file.Paths;
+
+/**
+ * Periodic consent job scheduler class.
+ * This class initialize the scheduler and schedule configured jobs and triggers.
+ */
+public class PeriodicalConsentJobScheduler {
```

Comment:
Shouldn't we use `java.util.concurrent.ScheduledExecutorService`?

Reply:
One of the main usages of the quartz is to support cron expressions and also to support task scheduling and running even in a clustered setup